### PR TITLE
Make letha the codeowner on the LR files

### PR DESCRIPTION
**Pull request by tas50**
This way we get reviews on potential docs changes before they make it into the docs. This reduces followup PRs and wasted effort.

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### mondoohq/docs CODEOWNERS
+
+# All resource updates require editorial review by Docs Editor Letha
+*.lr letha@mondoo.com
```
Comment (chris-rock) [inline .github/CODEOWNERS]: We need to define a deputy in case Letha is not available

Comment (tas50) [reply .github/CODEOWNERS]: I can do that if she's not around

**Comment by benr** [comment]
Updated to use a Github Team after discussion with @chris-rock.  This will be non-enforced and thus only a recommendation for review, not a requirement.

**Review by chris-rock — APPROVED** [review]
Thank you @benr

**Pull request merged**